Makefile

```diff
@@ -44,14 +42,12 @@ build-local:
 
 push:
 	@$(CONTAINER_ENGINE) push $(CONSOLE_IMAGE):latest
-	@$(CONTAINER_ENGINE) push $(CONSOLE_IMAGE):${VERSION}
 	@$(CONTAINER_ENGINE) push $(CONSOLE_IMAGE):${IMAGE_TAG}
 
-
-start-dev: export REACT_APP_CIQ_API_URL = http://localhost:8081
+start-dev: export VITE_CIQ_API_URL = http://localhost:8081
 start-dev:
 	@echo "### [Starting project DEV MODE] ###"
-	@echo "### [API-URL: $$REACT_APP_CIQ_API_URL] ###"
+	@echo "### [API-URL: $$VITE_CIQ_API_URL] ###"
 	@npm run start
```

Comment:
This is correct for running the console on local.
If we want to adapt this to run as a container (like the backend) we should replace this by a podman/docker command. 

As the backend is deployed using compose files, we need to deploy the console container attached to the compose network


I tried it by my own, and including these changes should fix this:
```
start-dev: export BACKEND_URL = http://api:8080
start-dev:
  @echo "### [Starting project DEV MODE] ###"
  @echo "### [API-URL: $$BACKEND_URL] ###"
  @podman run -dt --name console --network docker-compose_default -e BACKEND_URL=$$BACKEND_URL -p 8080:8080 quay.io/ecosystem-appeng/cluster-iq-console:latest
```

Reply:
It seems that a separate Makefile target is necessary to launch the frontend specifically in the container. 
I would reserve `start-dev` exclusively for launching without a container.